### ISSUE TITLE
Verify screenreader/OS/browser combo issues from r1 Fable testing

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/membrane-transport#399]
From Fable Round 1 Compatibility testing:

1. win/JAWS/Firefox user was not receiving information about the solute amounts after adding/removing.

> When I open the pop-up to adjust the amount it is not giving me audio feedback . I was able to adjust the amount but was not sure how much until I closed the item.

2. mac/Chrome/VoiceOver user was unable to find the Reset Solutes button. 

> I'm not able to locate a button or other type of element that allows me to remove the solutes. 

3. win/NVDA/Chrome could not read anything or was receiving no responses? 

> I was unable to note any simulation changes as my screen reader couldn't see them.

**Comment by brettfiedler** [comment]
1. I am hearing the context/object responses, but I am having the same trouble as NVDA in not being able to leave Forms mode when on the solute adjusters. This may have cropped up for the JAWS user.

**Comment by brettfiedler** [comment]
2. I don't have the devices for this. I will leave this for QA (note: @terracoda has tested this extensively, but not on newer macOS versions)

3. This works fine for me, so I am not sure the problem. I leave it for QA.

**Comment by brettfiedler** [comment]
Add to test in #338

**Comment by Nancy-Salpepi** [comment]
> 2. mac/Chrome/VoiceOver user was unable to find the Reset Solutes button.
I'm not able to locate a button or other type of element that allows me to remove the solutes.

I had no issue finding the button with mac 15.6 + safari 18.6/chrome latest.
